### PR TITLE
improve error alerts in versions

### PR DESCRIPTION
**Pull request by allanchau**
This PR improves the readability of error alerts in the versions plugin.

**Setup**

- [x] Add this to a project.

**Testing**

- [x] Generate an error (not a csrf one) an make sure the alert is different from the generic one.

### CODE REVIEW

File: lib/api/error.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+ */
+function store (err, req) {
+
+    req.linz = req.linz || {};
+    req.linz.error = err;
+
+}
+
+module.exports = {
+    json: json,
```
Comment (allanchau) [inline lib/api/error.js]: Can this just be es6?
```
module.exports = {
    json,
    store,
};
```

File: lib/api/error.js [hunk 2]
```diff
@@ -0,0 +1,34 @@
+'use strict';
+
+/**
+ * Set a flag to denote this error should be returned via JSON.
+ * @param  {Error} err         The error to decorate.
+ * @param  {Number} statusCode The statusCode to return.
+ * @return {Error}             The decorate error object.
+ */
+function json (err, statusCode = 500) {
```
[the remark this answers is not in the document]
Comment (smebberson) [reply lib/api/error.js]: Will do, I was just trying to be consistent with other files, but there is no time like the present to make things modern.

File: lib/api/error.js [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+
+}
+
+/**
+ * Store an error that has occurred on req.
+ * @param  {Object} req A HTTP request object.
+ * @param  {Error} err  The error that occurred.
+ * @return {Void}
+ */
+function store (err, req) {
```
[the remark this answers is not in the document]
Comment (smebberson) [reply lib/api/error.js]: Same here.

File: lib/api/error.js (hunk 1 above)
Comment (smebberson) [reply lib/api/error.js]: Yep, done.

**Comment by smebberson** [comment]
@allanchau, I've added some documentation, could you please have a read of that?

**Pull request merged**